packages/frontend-2/server/api/v2/healthz/liveness.get.ts

```diff
@@ -0,0 +1,5 @@
+export default defineEventHandler(() => {
```

Comment:
I'm not sure of the `/api/v2` prefix for fe2 API routes. It kind of implies that these are a v2 REST API of the `server` routes. Better to use a completely different prefix altogether like `/api-frontend` or something.